charts/cluster/templates/_bootstrap.tpl

```diff
@@ -7,6 +7,9 @@ bootstrap:
             {{- . | toYaml | nindent 4 }}
```

Comment:
Can you add `owner` to the `omit` statement above, so the section is not duplicated?